packages/expo-facebook/android/src/main/java/expo/modules/facebook/FacebookPackage.kt

```diff
@@ -0,0 +1,10 @@
+package expo.modules.facebook
+
+import android.content.Context
+import expo.modules.core.BasePackage
+import expo.modules.core.ExportedModule
+
+class FacebookPackage : BasePackage() {
+  override fun createExportedModules(context: Context): List<ExportedModule> =
+    listOf(FacebookModule(context) as ExportedModule)
```

Comment:
```suggestion
    listOf(FacebookModule(context))
```
Explicit cast should not be necessary as return type is explicitly specified (? something to verify)